core/server/worker/src/main/java/alluxio/worker/block/BlockLockManager.java

```diff
@@ -197,22 +197,20 @@ public void unlockBlock(long lockId) throws BlockDoesNotExistException {
    *
    * @param sessionId the session id
    * @param blockId the block id
-   * @throws BlockDoesNotExistException if the block id cannot be found
+   * @return false if it failed to unlock due to lock not found
    */
   // TODO(bin): Temporary, remove me later.
-  public void unlockBlock(long sessionId, long blockId) throws BlockDoesNotExistException {
+  public boolean unlockBlock(long sessionId, long blockId) {
```

Comment:
This is marked as a temporary method, should we be improving the permanent method?